settings/LoanPolicyForm.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Field, getFormValues } from 'redux-form';
-import _ from 'lodash';
-
 import { FormattedMessage } from 'react-intl';
+import _ from 'lodash';
```

Comment:
@anton-plotnikov would it be ok if you just import what you need from lodash instead of the full `_` ( I think we can do the same in `LoanPolicyDetail.js`)? This should help with tree shaking in the future.